servers/Azure.Mcp.Server/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ The Azure MCP Server updates automatically by default whenever a new release com
 
 ### Bugs Fixed
 
+- Fixes issue where client options not passed to ConfigurationClient [[#44](https://github.com/microsoft/mcp/pull/44)]
```

Comment:
This sentence is not context free and could be better for our end users (and especially our support team who reads these, looking to see if a change will help their customers). 

2 pieces of context would help 1) this change is specific only to AppConfig; 2) this change is specific to Telemetry

So I'd say something like "Fix a bug with Telemetry collecting related to AppConfig MCP tools"

Reply:
updated